setup.py

```diff
@@ -53,7 +53,7 @@
         'tzlocal',
         'requests',
         'certifi',
-        'importlib_metadata',
+        "importlib_metadata;python_version<'3.8'",
```

Comment:
This is not legal syntax here, nor do I think it is required for the desired outcome. Probably should just revert this. 

What really should happen is the entire `requires` block needs to be removed, as it is basically deprecated and its use with `install_requires` is not recommended. This is really just a legacy appendage that needs pruning. I'd prefer it if you reverted it and then I will take out the entire block after merging.